contracts/Parameterizer.sol

```diff
@@ -113,6 +113,12 @@ contract Parameterizer {
 
         // type of majority out of 100 necessary for proposal success in parameterizer
         set("pVoteQuorum", _parameters[11]);
+
+        // minimum length of time between initExit & finalizeExit
+        set("exitTimeDelay", _parameters[12]);
+
+        // maximum length of time between initExit & finalizeExit
+        set("exitTimeExpiry", _parameters[13]);
```

Comment:
I find these parameter names `exitTimeDelay` and `exitTimeExpiry` to be somewhat unclear. My understanding is that these are being used to calculate the bounds of an exit period. The delay means you can't exit immediately. The expiry means that there is a point after which you can no longer exit unless you call `initExit` again.

```
   |  delay  |   exitPeriodLen   |
   ^                     ^
  init                finalize somewhere in here
```
I think it would be clearer to use a parameter for the length of this period, e.g. `exitPeriodLen` or `exitStageLen` instead.